README.md

```diff
@@ -15,50 +15,54 @@ npm install --save-dev @enact/jsdoc-to-ts
 Assuming this directory is a peer of the Enact source and you want to write this into an installed (from npm) version of enact:
 
 ```bash
-node -e "['core', 'ui', 'moonstone', 'i18n', 'webos', 'spotlight'].forEach(p => require('.')({
-  package: '../enact/packages/' + p,
-  output: require('fs').writeFileSync,
-  ignore: ['node_modules', 'ilib'],
-  importMap: {
-    ui: '@enact/ui',
-    moonstone: '@enact/moonstone',
-    core: '@enact/core',
-    webos: '@enact/webos',
-    spotlight: '@enact/spotlight',
-    i18n: '@enact/i18n'
-  },
-  outputPath: '<path to installation>/node_modules/@enact/' + p
-}))"
+node -e "import('./index.js').then(({default: jsdocToTs}) => {
```

Comment:
I think we need to mention using `import` from which version of the node is possible.
What do you think?

Reply:
You're right. Let's add the required node version.